enerdata/profiles/profile.py

```diff
@@ -381,6 +417,10 @@ def estimate(self, tariff, balance):
             cofs = self.profile_class.get_range(
                 start, end - relativedelta(days=1)
             )
+
+        if tariff.code == 'RE':
+            cofs = self.profile_class.get_RE_cofs(start, end, tariff)
+
```

Comment:
Don't do this, this is thought to assign `profile_class` and you should have a class with the necessary methods.

Reply:
done via: https://github.com/gisce/enerdata/pull/115/commits/de579f721e5db56c10a82841c560d3fa1a935eb2, https://github.com/gisce/enerdata/pull/115/commits/e98b1c8755d743efbf0e122989f491d6216f1b34, https://github.com/gisce/enerdata/pull/115/commits/0d9c1824ac0da38115a8cdb0dddfe593b9b61147, https://github.com/gisce/enerdata/pull/115/commits/31e98b5fd941b6e87e1e477e11fae7937e801508, https://github.com/gisce/enerdata/pull/115/commits/5a6a4bc1aa938a9a9d586c32b09b7f1660b28c07